### PR TITLE
Update User Guide with features: listo and findo

### LINKED ISSUE #22: Update User Guide (Janelle)

### PR DESCRIPTION
**Pull request by janelleljt**

### CODE REVIEW

**Review by sohenze — APPROVED** [review]
Looks good

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,192 +1,31 @@
...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
-
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+### Locating persons by name: `findo`
```
Comment (donljh) [inline docs/UserGuide.md]: I think you might have forgotten to change 'Locating persons by name' to the relevant title here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,192 +1,31 @@
...
 
-### Locating persons by name: `find`
-
-Finds persons whose names contain any of the given keywords.
+Find order with item names containing any of the given keywords.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
```
Comment (donljh) [inline docs/UserGuide.md]: Command here should be 'findo' instead of 'find'

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,192 +1,31 @@
...
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* The search is case-insensitive. e.g `keychain` will match `Kans`
```
Comment (donljh) [inline docs/UserGuide.md]: I think you meant to change 'Hans' to 'Keychain' instead of 'Kans' here

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,192 +1,31 @@
...
+* The order of the keywords does not matter. e.g. `apple keychain` will match `Keychain Apple`
 * Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
+* Only full words will be matched e.g. `keychains` will not match `keychain`
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (donljh) [inline docs/UserGuide.md]: 'Persons' should be 'Orders with item names' instead

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,192 +1,31 @@
...
---------------------------------------------------------------------------------------------------------------------
-
-## FAQ
-
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
-
---------------------------------------------------------------------------------------------------------------------
+* `find keychain` returns `banana keychain` and `keychain`
+* `find apple keychain` returns `apple painting`, `banana keychain`<br>
```
Comment (donljh) [inline docs/UserGuide.md]: Like above, this should be 'findo' instead of 'find'

File: docs/UserGuide.md [hunk 6]
```diff
@@ -1,192 +1,31 @@
...
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+**listo**| `listo`
+**findo** | `find KEYWORD [MORE_KEYWORDS]`
```
Comment (donljh) [inline docs/UserGuide.md]: 'findo' instead of 'find' here as well

**Review by donljh — CHANGES_REQUESTED** [review]

**Review by donljh — APPROVED** [review]

**Pull request merged**